nixos/modules/programs/xastir.nix

```diff
@@ -0,0 +1,23 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.programs.xastir;
+in {
+  meta.maintainers = with maintainers; [ melling ];
+
+  options.programs.xastir = {
+    enable = mkEnableOption (mdDoc "Enable Xastir Graphical APRS client");
```

Comment:
mkEnableOption prefixes the string. It looks now like: ``Whether to enable Enable Xastir Graphical APRS client.``

Reply:
Thanks & apologies Sandro. How should I recitfy this, another PR?